src/support.cpp

```diff
@@ -500,8 +500,8 @@ Polygons AreaSupport::join(const SliceDataStorage& storage, const Polygons& supp
                 break;
             case EPlatformAdhesion::RAFT:
             {
-                const ExtruderTrain& raft_extruder = mesh_group_settings.get<ExtruderTrain&>("adhesion_extruder_nr");
-                adhesion_size = raft_extruder.settings.get<coord_t>("raft_margin");
+                const ExtruderTrain& raft_base_extruder = mesh_group_settings.get<ExtruderTrain&>("raft_base_extruder_nr");
+                adhesion_size = raft_base_extruder.settings.get<coord_t>("raft_margin");
```

Comment:
The `raft_margin` setting gets set to `limit_to_extruder = adhesion_extruder_nr`, so you'd expect `adhesion_extruder_nr` to be the correct extruder?